/**:
  ros__parameters:
    encryption_key: "germanopen2019"
    port: 1234
    send_frequency: 10
    sender_queue_max_size: 5
    target_ips:
      - 172.20.2.17

    topics:
      - /rosout
      - /robot_state
      - /cmd_vel
      - /debug/dsd/body_behavior
      - /debug/dsd/head_behavior
      - /debug/dsd/hcm
      - /team_data
      - /balls_relative
      - /ball_relative
      - /robots_relative
      - /gamestate
      - /animation
      - /diagnostics_agg
      - /joint_states 
      - /imu/data
      - /buttons
      - /system_workload
